backend/app.js

```diff
@@ -13,33 +13,33 @@ const userId = process.env.userId;
 const basicAuth = process.env.basicAuth;
 const googleSheet = process.env.googleSheet;
 const readline = require("readline");
-const {google} = require("googleapis");
+const { google } = require("googleapis");
 var fs = require("fs");
 const ordersdb = require("./orderquery");
 const bcryptjs = require("bcryptjs");
 app.use(cors());
 app.use(bodyParser.json());
 const withAuth = require("./middleware");
+const checkAccess = require("./withAccess");
```

Comment:
the naming on this is a little weird, can you rename the file to checkAccess as well?